cirq/circuits/circuit.py

```diff
@@ -66,6 +67,14 @@
 INT_TYPE = Union[int, np.integer]
 
 
+class Alignment(enum.Enum):
```

Comment:
We have `cirq.AlignLeft` and `cirq.AlignRight` optimizers. 
1. Do you think it would be useful to rename these `LEFT` and `RIGHT`? (although this assumes no transposing) If not, do you think we should rename those optimizers as well? 
2. I have this inkling that in the cirq namespace, we don't know the context of this generic term "alignment" - maybe we should name this `cirq.CircuitAlignment`?

Reply:
Changed to `LEFT` and `RIGHT`. I think it makes sense to match that terminology. I think I'll leave it as `cirq.Alignment` to match `cirq.AlignLeft` and `cirq.AlignRight` which are similarly generic terms (we could even have a factory function that takes an `Alignment` and returns the appropriate optimizer if we wanted).